reconcile/utils/saasherder.py

```diff
@@ -388,13 +388,38 @@ def _process_template(self, options):
                 return None, None
 
             if "IMAGE_TAG" not in consolidated_parameters:
-                template_parameters = template.get('parameters')
-                if template_parameters is not None:
-                    for template_parameter in template_parameters:
-                        if template_parameter['name'] == 'IMAGE_TAG':
-                            # add IMAGE_TAG only if it is required
-                            image_tag = commit_sha[:hash_length]
-                            consolidated_parameters['IMAGE_TAG'] = image_tag
+                for template_parameter in template.get("parameters", {}):
```

Comment:
This delta not strictly necessary, just saved an indent and made the two chunks match stylistically.

Reply:
Nope. The second argument to `.get()` is the default value to use if the key is not present. The default default is `None`, but here I switched to use `{}` (the empty dict) which will make the loop iterate zero times.

Reply:
Reverted.